test/tpu/pjrt_test_job.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: pjrt-test-job-
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: tpu.googleapis.com/name
+            operator: In
+            values:
+            # Update with your TPU name
+            - gke-tpu
```

Comment:
Do you need the `name` constraint here? These tests should work on any TPU with the correct `type`

Reply:
Removed